packages/gatsby-cli/src/structured-errors/error-map.ts

```diff
@@ -641,7 +641,17 @@ const errors = {
     text: ({ inputManifest, pagePath }): string =>
       `${getSharedNodeManifestWarning(
         inputManifest
-      )} but Gatsby didn't find a ownerNodeId for the page at ${pagePath}\nUsing the first page that was found with the node manifest id set in pageContext.id in createPage().\nThis may result in an inaccurate node manifest (for previews or other purposes).`,
+      )} but Gatsby didn't find an ownerNodeId for the page at ${pagePath}\nUsing the first page that was found with the node manifest id set in pageContext.id in createPage().\nThis may result in an inaccurate node manifest (for previews or other purposes).`,
+    level: Level.WARNING,
+    category: ErrorCategory.USER,
+  },
+
+  "11805": {
+    // @todo add docs link to "using Preview" once it's updated with an explanation of ownerNodeId
```

Comment:
I know this is just copied and pasted from the other log message, but don't we have an explanation of ownerNodeId that we can link to now?

Reply:
Oh good call! I'll change the message in a sec.
This solution isn't specific to Contentful, it'll work for anyone who makes GraphQL queries using a slug to find the right node. I just know a lot of Contentful sites do this since the main starter works that way, so it's most meaningful for us for Contentful sites even if it'll work generally, since Contentful is our #1 integration as far as current userbase goes.
`ownerNodeId` is the way to be 100% certain, but it's a lot easier to add these kinds of improvements than it is to get everyone to change their site code manually, so we should be doing both 👍